plugins/woocommerce/includes/class-woocommerce.php

```diff
@@ -1203,4 +1205,19 @@ public function get_instance_of( string $class_name, ...$args ) {
 	public function get_global( string $global_name ) {
 		return wc_get_container()->get( LegacyProxy::class )->get_global( $global_name );
 	}
+
+	/**
+	 * Converts the WooCommerce slug to the correct slug for the current version.
+	 * This ensures that when the plugin is installed in a different folder name, the correct slug is used so that dependent plugins can be installed/activated.
+	 *
+	 * @param string $slug The plugin slug to convert.
+	 *
+	 * @return string
+	 */
```

Comment:
Can we add a `@since` tag to this block?

Reply:
Yep, added in 6dbe049c20cab7864f3faaeddf877454bd78f10e. Thanks!